CONTRIBUTING.md

```diff
@@ -57,3 +57,20 @@ new code.
    directly onto the `package.json` (NPM) version.
  - Releases are also tagged so they are present on the Releases page on Github.
  - Releases should be signed by the maintainer's key.
+
+### Release process
```

Comment:
With the `release-` (feature-freeze) branches, I think the commands for a release candidate are more helpful than those for a final release.

Reply:
the process for an RC and a release are identical except that for a main release we merge back into master at the end, and flip a bit on the GitHub release. Rather than duplicating these instructions, I'd just add an appendix for doing a full release.